plot/README.md

```diff
@@ -14,18 +14,38 @@ Each plot will have the following parameters:
 1. plot_title: Title to be displayed above the line plot.
 2. x_axis_label: Label to be displayed along the x-axis.
 3. y_axis_label: Label to be displayed along the y-axis.
-4. plot_type: The type of plot to be displayed i.e `line`, `bar`, `dot`, `area`, `dot-lines`.
-5. traces: Contains the information about each each trace.
+4. plot_type: The type of plot to be displayed i.e `line`, `bar`, `dot`, `area`, `dot-lines`, `pie`,  `histogram-horizontal`, `histogram-verical`.
+5. config:  the config for the plot
+      1. bargap: Distance between the bins in the histogram - Only valid for plot type as histogram
+      2. showlegend: To show the legend or not
+      3. height: Height of the plot // Default: 500
+      4. width: width of the plot // Deault: 1000
+      5. slice_label: label to be shown on slcie of a pie - oly for pie chart // Default: "none", Available: "percent" or "value"
+Traces can be based on 1d and 2d plot.
+For 2d plot,
+6. traces: Contains the information about each each trace.
       1. uri : The url from which the data has to be fetched.
       2. legend: The value of legend to be shown for this trace.
       3. x_col: The column name for the x values
       4. y_col: An array of column name for the y values
-6. plotlyButtonsToRemove: The button to be removed shown by plotly by defualt.
+
+For 1d plot,
+6. traces: Contains the information about each each trace.
+      1. uri : The url from which the data has to be fetched.
+      2. legend: The value of legend to be shown for this trace.
+      3. data_col: The column name for the x values
+      4. legend_col: An array of column name for the y values
```

Comment:
Is this meant to be the title for the y axis or an enumeration of the value labels for the y axis?